ui/app/routes/vault/cluster.js

```diff
@@ -137,6 +140,32 @@ export default Route.extend(ModelBoundaryRoute, ClusterRoute, {
     if (this.namespaceService.path && !this.version.hasNamespaces) {
       return this.router.transitionTo(this.routeName, { queryParams: { namespace: '' } });
     }
+
+    // identify user for analytics service
+    if (this.analytics.activated) {
+      try {
+        let license;
+        const token = await this.api.auth.tokenLookUpSelf();
+        const entity = token.data.entity_id ? token.data.entity_id : `root_${crypto.randomUUID()}`;
+
+        if (model.license?.state) {
```

Comment:
License is only available to for ent customers, so do some guarding in case this somehow gets run in CE mode

Reply:
The whole `model.license` is undefined. Which makes sense

Reply:
Ah okay, I was wondering if this should be `mode?.license?.state` then